src/controllers/commendations.ts

```diff
@@ -57,14 +57,18 @@ const create = async (req: Request, res: Response) => {
     newCommendation._id = uuidv4();
     newCommendation.date = new Date().toISOString();
 
+    let isTeamCommendation = () => {
+        return (req.query.isTeamCommendation === "true")
```

Comment:
Hm. Why make it a function if it's scoped to here anyways? You can inline this.

Reply:
Also, whenever we declare functions, we declare them as `const` since they can never be changed. Good job understanding the syntax, though.